presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -349,6 +349,9 @@ private boolean isColumnsEnumeratingTable(SchemaTableName schemaTableName)
         Optional<Set<String>> tables = filterString(constraint, TABLE_NAME_COLUMN_HANDLE);
         if (tables.isPresent()) {
             return prefixes.stream()
+                    .flatMap(prefix -> prefix.getSchemaName()
+                            .map(schemaName -> Stream.of(prefix))
+                            .orElseGet(() -> listSchemaNames(session)))
                     .flatMap(prefix -> tables.get().stream()
```

Comment:
this is no longer `prefix` but `schema`

Reply:
technically it's still prefix, see `prefix.getSchemaName()` below